docs/kiali/05-01-kiali.md

```diff
@@ -21,4 +21,4 @@ This table lists the configurable parameters, their descriptions, and default va
 | **deployment.accessibleNamespaces** | Specifies the Namespaces Kiali can access to monitor the Service Mesh components deployed there. You can provide the names using regex expressions. The default value is `**`(two asterisks) meaning Kiali can access any Namespace. | `**` |
 
 
-For details on Kiali configuration and customization, see [this](https://www.kiali.io/documentation/getting-started/) documentation.
+For details on Kiali configuration and customization, see [this](https://www.kiali.io/documentation/getting-started/) documentation.
```

Comment:
I think it is more important to link the CRD documentation at https://github.com/kiali/kiali-operator/blob/master/deploy/kiali/kiali_cr.yaml or simply the values.yaml of kyma https://github.com/kyma-project/kyma/blob/master/resources/kiali/values.yaml

Reply:
Changed this line, now links to both the CRD and the values files on the kyma repository.